src/anomalib/deploy/inferencers/openvino_inferencer.py

```diff
@@ -69,12 +69,19 @@ class OpenVINOInferencer(Inferencer):
         metadata is loaded from the ``metadata.json`` file. To make a prediction,
         we can simply call the ``predict`` method:
 
-        >>> import cv2
-        >>> image = cv2.imread("path/to/image.jpg")
-        >>> image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
-        >>> result = inferencer.predict(image)
+        >>> prediction = inferencer.predict(image="path/to/image.jpg")
 
-        ``result`` will be an ``ImageResult`` object containing the prediction
+        Alternatively we can also pass the image as a PIL image or numpy array:
+
+        >>> from PIL import Image
```

Comment:
Maybe you could add 1 more example where we directly pass the path to the image to the `predict` method.